apps/api/src/app/environments-v1/usecases/output-renderers/in-app-output-renderer.usecase.ts

```diff
@@ -7,7 +7,7 @@ import { RenderCommand } from './render-command';
 export class InAppOutputRendererUsecase {
   @InstrumentUsecase()
   execute(renderCommand: RenderCommand): InAppRenderOutput {
-    const { skip, ...outputControls } = renderCommand.controlValues ?? {};
+    const { skip, disableOutputSanitization, ...outputControls } = renderCommand.controlValues ?? {};
```

Comment:
looks like the framework trims the additional params in json schema validation, but we should not rely on this and pass unexpected data.